cni-plugin/pkg/install/install.go

```diff
@@ -117,6 +119,9 @@ func loadConfig() config {
 }
 
 func Install() error {
+	// Configure logging before anything else.
+	logrus.SetFormatter(&logutils.Formatter{Component: "cni-installer"})
+
 	// Clean up any existing binaries / config / assets.
 	if err := os.Remove("/host/opt/cni/bin/calico"); err != nil && !os.IsNotExist(err) {
```

Comment:
Still got the blanket `Remove()` up here also this bit doesn't seem to be aware of the multiple install directories

Reply:
Doh, I was looking for that and for whatever reason couldn't find it and said "I guess we don't actually delete the binaries"